include/deal.II/physics/notation.h

```diff
@@ -0,0 +1,619 @@
+     *  v_{0} & v_{1} & v_{2}
+     * \end{array}\right]^{T}
+     * \quad \Rightarrow \quad
+     * \left[ \begin{array}{ccc}
+     *  n = 0 & n = 1 & n = 2
+     * \end{array}\right]^{T} .
+     * @f]
+     * To summarise, the relationship between tensor and Kelvin indices for both
+     * the three-dimensional case and the analogously discerned two-dimensional
+     * case outlined in the following table:
+     * <table>
+     * <tr>
+     *   <th align="center"> Dimension 2 </th>
+     *   <th align="center"> Dimension 3 </th>
+     * </tr>
+     * <tr>
+     * <td align="middle">
+     *   <table>
+     *   <tr>
+     *     <th>Tensor index pairs</th>
+     *     <th>Kelvin index</th>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">00</td>
+     *     <td align="center">0</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">11</td>
+     *     <td align="center">1</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">12</td>
+     *     <td align="center">2</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">21</td>
+     *     <td align="center">3</td>
+     *   </tr>
+     *   </table>
+     * </td>
+     * <td align="middle">
+     *   <table>
+     *   <tr>
+     *     <th>Tensor index pairs</th>
+     *     <th>Kelvin index</th>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">00</td>
+     *     <td align="center">0</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">11</td>
+     *     <td align="center">1</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">22</td>
+     *     <td align="center">2</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">12</td>
+     *     <td align="center">3</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">02</td>
+     *     <td align="center">4</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">01</td>
+     *     <td align="center">5</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">10</td>
+     *     <td align="center">6</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">20</td>
+     *     <td align="center">7</td>
+     *   </tr>
+     *   <tr>
+     *     <td align="center">21</td>
+     *     <td align="center">8</td>
+     *   </tr>
+     *   </table>
+     * </td>
+     * </tr>
+     * </table>
+     *
+     * To illustrate the purpose of this notation, consider the rank-2 symmetric
+     * tensors $\mathbf{S}$ and $\mathbf{E}$ that are related to one another by
+     * $\mathbf{S} = \cal{C} : \mathbf{E}$, where the operator $\cal{C}$ is a fourth-order
+     * symmetric tensor.
+     * As opposed to the commonly used Voigt notation, Kelvin (or Mandel) notation
+     * keeps the same definition of the inner product $\mathbf{S} : \mathbf{E}$
+     * when both $\mathbf{S}$ and $\mathbf{E}$ are symmetric. In general the inner product
+     * norms of all symmetric and general tensors remain the same.
```

Comment:
what is "inner product norm"?

Reply:
As per our discussion, I'll remove "norm", and qualify that "remain the same" refers to the properties of the normal and flattened tensors.